pkg/services/live/push/push.go

```diff
@@ -28,14 +40,24 @@ type Gateway struct {
 	GrafanaLive *live.GrafanaLive `inject:""`
 
 	converter *convert.Converter
+
+	// TODO: need to periodically invalidate this cache in separate goroutine.
+	keyMu    sync.RWMutex
+	keyCache map[string]CachedKey
+}
+
+func NewGateway() *Gateway {
+	return &Gateway{
+		keyCache: map[string]CachedKey{},
+	}
 }
 
 // Init Gateway.
 func (g *Gateway) Init() error {
-	logger.Info("Telemetry Gateway initialization")
+	logger.Info("Live Push Gateway initialization")
```

Comment:
This will be logged even if it's not enabled so prefer debug.

```suggestion
	logger.Debug("Live Push Gateway initialization")
```